cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/transform/LogParserTransform.java

```diff
@@ -82,10 +82,14 @@
   private static final int S3_REGEX_LENGTH = 18;
   //Number of groups matched in the CLF regex
   private static final int CLF_REGEX_LENGTH = 9;
+  //Number of fields in a Cloudfront log
+  private static final int CLOUDFRONT_LENGTH = 23;
   private static final String S3_LOG = "S3";
   private static final String CLF_LOG = "CLF";
+  private static final String CLOUDFRONT_LOG = "Cloudfront";
```

Comment:
You don't have to catch ``enum.valueOf(String)`` - it throws unchecked exception? Anyway, I think its ok to leave it for now.